examples/registration.py

```diff
@@ -0,0 +1,27 @@
+"""
+Load and show sample data
```

Comment:
This example is called `registration`. Ideally it should also carry out the registration, or if not be called something like `load_sample_data.py`.

Reply:
Changed to `load_sample_data.py`.